### PR TITLE
Add image syntax docs

### PR DESCRIPTION
**Pull request by dannytranlx**
^ title

@jpjoyal 
@mspensieri 
@jugarrit 
@Mario54

### CODE REVIEW

File: source/includes/_images.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Images
+
+You can send your users an image from the web with the following syntax:
```
Comment (mspensieri) [inline source/includes/_images.md]: "an image from the web" is a bit confusing to me. What about this?

> To send the user an image from one of your Business System integrations, use the following syntax with a publicly visible web url

Comment (dannytranlx) [inline source/includes/_images.md]: Yeah, that's what I meant 😂  Will update!

**Comment by chloepouprom** [comment]
👍

**Comment by mspensieri** [comment]
👍

**Pull request merged**